nautobot/core/management/commands/generate_performance_test_endpoints.py

```diff
@@ -27,15 +49,17 @@ def handle(self, *args, **options):
             # De-duplicate the URL patterns and sort them.
             self.app_name_to_urls["endpoints"][view_name] = sorted(list(set(url_patterns)))
 
-        with open("./endpoints.yml", "w") as outfile:
-            yaml.dump(self.app_name_to_urls, outfile, sort_keys=True)
+        self.stdout.write(yaml.dump(self.app_name_to_urls, sort_keys=True))
```

Comment:
I'd expect it to dump to stdout only if there is *not* an output_file specified?